NOTICE-executable

```diff
@@ -0,0 +1,637 @@
+Embulk
+Copyright 2014-2023 Sadayuki Furuhashi, and the Embulk project.
+
+This product includes software developed by
+Sadayuki Furuhashi, and the Embulk project (https://www.embulk.org/).
+
+
+
+The executable distribution of this product includes
+the API for SLF4J (The Simple Logging Facade for Java), developed by
+QOS.ch Sarl (Switzerland), licensed under
+the MIT License (http://www.opensource.org/licenses/mit-license.php).
+
+* Copyright (c) 2004-2022 QOS.ch Sarl (Switzerland)
+
+
+
+The executable distribution of this product includes
+Logback, developed by QOS.ch, dual-licensed under
+the Eclipse Public License v1.0, or the GNU Lesser General Public License version 2.1.
+
+* Copyright (C) 1999-2015, QOS.ch. All rights reserved.
+
+Note that this product adopts the Eclipse Public License v1.0 for Logback.
+Refer to the license of Logback at META-INF/LICENSE-logback.
+
+
+
+The executable distribution of this product includes
+MessagePack for Java, licensed under
+the Apache License, Version 2.0.
+
+* The NOTICE file of MessagePack for Java is embedded in this NOTICE below.
+
+
+
+The executable distribution of this product includes
+Jackson JSON processor, developed by
+Tatu Saloranta, tatu.saloranta@iki.fi, licensed under
+the Apache License, Version 2.0.
+
+* Copyright (c) 2007- Tatu Saloranta, tatu.saloranta@iki.fi
+* The NOTICE file of Jackson JSON processor is embedded in this NOTICE below.
+
+
+
+The executable distribution of this product includes
+Slice, developed by the Airlift project, licensed under
+the Apache License, Version 2.0.
+
```

Comment:
Missing copyright?

Reply:
I imported copyright notations from the original declarations. I didn't include copyright notations if I was not able to find it in the original source code repositories nor distributions.